src/Umbraco.Web.UI.Client/src/views/propertyeditors/slider/slider.controller.js

```diff
@@ -34,6 +41,76 @@
     else {
         $scope.model.config.step = parseFloat($scope.model.config.step);
     }
+
+    if (!$scope.model.config.handle) {
+        $scope.model.config.handle = "round";
+    }
+
+    if (!$scope.model.config.reversed) {
+        $scope.model.config.reversed = false;
+    }
+    else {
+        $scope.model.config.reversed = $scope.model.config.reversed === "1" ? true : false;
+    }
+
+    if (!$scope.model.config.tooltip) {
+        $scope.model.config.tooltip = "show";
+    }
+
+    if (!$scope.model.config.tooltipSplit) {
+        $scope.model.config.tooltipSplit = false;
+    }
+    else {
+        $scope.model.config.tooltipSplit = $scope.model.config.tooltipSplit === "1" ? true : false;
+    }
+
+    if ($scope.model.config.tooltipFormat) {
+        $scope.model.config.formatter = function (value) {
+            if (angular.isArray(value) && $scope.model.config.enableRange) {
+                return $scope.model.config.tooltipFormat.replace("{0}", value[0]).replace("{1}", value[1]);
+            } else {
+                return $scope.model.config.tooltipFormat.replace("{0}", value);
+            }
+        }
+    }

```

Comment:
```$scope.model.config.formatter``` is used to returns a string for tooltips instead of just the value. In the slider config you can use ```{0}``` and ```{1}``` for the values. ```{1}``` is only when it is a range slider and when tooltipSplit is false.

When tooltipSplit is false it use one tooltip for range slider in the format, ```0 : 100```, but with this option you can write show ```$0 - $100```, by using ```${0} - ${1}```.